distribution/lib/Standard/Base/0.0.0-dev/src/Function.enso

```diff
@@ -95,7 +100,8 @@ flip f = (x -> y -> f y x)
    - x constant value to return
 
    > Example
-    IO.println <| [1, 2, 3].map (Function.const 7) # Prints '[7, 7, 7]'
+
+       IO.println <| [1, 2, 3].map (Function.const 7) # Prints '[7, 7, 7]'
```

Comment:
I'm confused. Sometimes we have to replace `<` with `&lt;` but sometimes apparently not. What is the rule here?

Reply:
Examples are picked up and encoded as code blocks in indented by 4 spaces.

Not true if in plain text section at moment.

Reply:
Ah I see so that is the _rule_ I wasn't sure about. If example blocks are not affected that does not seem as bad as I was worried.